src/components/pages/Profile/Profile.js

```diff
@@ -0,0 +1,62 @@
+/* global gapi */
+import React, { Component } from 'react';
+import {Image} from 'react-bootstrap';
+import './Profile.scss';
+
+class Profile extends Component {
+    constructor() {
+        super();
+
+        this.state = {
+            email: null,
+            profile:  null,
+            profiles: null
+        }
+    }
+
+    
+    loadProfiles(response) {
+        var jsonDoc = JSON.parse(response.result.response.result)
+        this.setState({profiles: jsonDoc});
+        var profile = jsonDoc.find((p)=> {
+            if(p.email === this.state.email) {
+                return true;
+            }
+            return false;
+        })
+        this.setState({profile: profile});
+    }
+
+    getProfiles() {
+        gapi.client.script.scripts.run({
+            'scriptId':  '16BTJrEFzzQoVRGdyAZzu0h1cf5bT6FxFb9wXwHJ-1nbFiTjLeIYek_hw', 
+            resource: {
+                function: 'getJsonDoc'
+            }
+        }).then(this.loadProfiles.bind(this));
```

Comment:
small thing: if you create class methods as arrow functions, you don't need to bind this. 

``getProfiles = () => { ... }``